### ISSUE TITLE
Tile sometimes permanently move when dragging

### ISSUE THREAD

**Issue by tslnox** [billthefarmer/gridle#10]
I've been experimenting but I haven't found any conditions on which that happens. Some stuff I found out:

- I can tell it happens more frequently on the large grid, and it looks like probability increases with more games played in a row. On small grid it happened only once, but I've played way more large grids.

- If I force kill the game and run it again, usually it stops happening for few first games. 

- Bottom tiles are more likely to move.

Device: Poco X3 NFC (Surya)
ROM: ArrowOS 12.1
Game version: 1.06 from Fdroid 

Thanks

**Comment by billthefarmer** [comment]
I don't understand what you mean by 'permanently move'. Do you mean that you can't move it again or that it makes a move while you are dragging it?

**Comment by tslnox** [comment]
The static "box" that contains letter moves out of the grid. I don't know why I didn't screenshot it. But it haven't happened since so I don't think we'll figure out why did it happen.

**Comment by billthefarmer** [comment]
What actually happens is that there is an extra slightly larger box normally parked outside the grid and set invisible. When a letter is moved this box is positioned on top, populated with the appropriate letter, set visible and moves with the original box. When the user lets go, the original box moved back to where it was, the letters are swapped, and the larger box made invisible. The reason for all this malarkey is that the boxes are actually stacked on top of each other so moving a box from the top makes it slide underneath the others and it gets clipped if moved outside the grid. This creates an illusion of swapping the boxes and the one being moved gets bigger to make it easier to see what is happening. This is not unusual, &ndash; most of android is an illusion to make users think they are doing something useful.

What may have happened is some sort of glitch that temporarily stopped some of that working until you did something else and it sorted itself out.
